package/batocera/emulationstation/batocera-es-system/es_systems.yml

```diff
@@ -1211,8 +1211,8 @@ openbor:
   extensions: [pak]
   emulators:
     openbor:
-      openbor: { requireAnyOf: [BR2_PACKAGE_OPENBOR] }
-      openbor4432: { requireAnyOf: [BR2_PACKAGE_OPENBOR4432] }
+      openbor3b6412: { requireAnyOf: [BR2_PACKAGE_OPENBOR] }
```

Comment:
The name must be the same as the binary

Reply:
No binary file is launched in by Python the name here is not really important.
It works fine for me like that.
But i you can rename also the binary files it's better for sure for future updates. I don't know where to do it.

Reply:
it works because you left commandArray = ["OpenBOR4432", rom] @jdorigao  is right

Reply:
yes for sure, i don't want to break it. But in menu it's really important to rename them. we must show we use the last OpenBOR version 3 build, the 6412.
Next time we will update this engine we will let v36412 like it is, and add the new OpenBOR version 4 build ??? that for now is in RC2.
And i wish Susan will finish to upgrade on SDL2 the old v3b3400 for all the old games.

After i'm agree with @jdorigao , it will be better to also rename the binary file on compilation, and maybe also the two ini files ...

But for now OpenBOR forced core is break and it's why i made this fix.

Reply:
i see now your pr makes sense, thanks for clarifying

Reply:
@jdorigao ok to merge?

Reply:
I've speak about it and explain to @susan yesterday on discord and now he has understand fine for OpenBOR. He write me he will merge it next week when he can. For now he work on Pacman.

I've verify OB 4.0 is only in Alpha:
https://github.com/DCurrent/openbor/releases